front/app/containers/Admin/settings/general/index.tsx

```diff
@@ -261,7 +261,7 @@ class SettingsGeneralTab extends PureComponent<
     }
   };
 
-  handleSettingChange = (settingName: TAppConfigurationSetting) => {
+  handleSettingChange = (settingName: TAppConfigurationSettingWithEnabled) => {
```

Comment:
Also a solution to deal with settings that need enabled. I think an 🆗 solution, because it solves our type issue and we're not going to have homepage settings here anyway.